### ISSUE TITLE
How to reproduce the results of finetune on ScanNet in Concerto

### ISSUE THREAD

**Issue by bkxmw** [Pointcept/Pointcept#531]
Thank you for your excellent work! I am currently trying to reproduce the fine-tuning results of Concerto on the ScanNet dataset, but the results I obtained show a noticeable gap compared to those reported in the paper.
My results: 0.7963 / 0.8669 / 0.9273
Reported results in the paper: 80.7 / 87.4 / 93.1
The command and configuration I used are as follows:
~~~
sh scripts/train.sh -g 8 -d concerto -c semseg-ptv3-large-v1m1-0c-scannet-ft -n 1031scannet_ft_concerto -w ./.cache/concerto/concerto_large.pth
~~~
I have confirmed that the pre-trained weights of encoder were loaded correctly.
~~~
Loading layer weights with keyword: module, replace keyword with: module.backbone
[2025-11-01 05:28:06,853 INFO misc.py line 262 3110475] Missing keys: ['seg_head.weight', 'seg_head.bias', 'backbone.dec.dec3.up.proj.0.weight', 'backbone.dec.dec3.up.proj.0.bias', 'backbone.dec.dec3.up.proj.1.weight',.......
~~~
What parameters or configurations should I adjust to achieve the results reported in the paper?

**Comment by Yujia-Zhang0913** [comment]
Hi, current default models(concerto_large/base/small/tiny) are the pre-release version of our next work, which adapts input without color and normal for more general use. The original version of Concerto can be found at [concerto_base_origin.pth](https://huggingface.co/Pointcept/Concerto/blob/main/concerto_base_origin.pth). 
You can use commands like 
```
sh scripts/train.sh -m 1 -g 8 -d sonata -c semseg-sonata-v1m1-0c-scannet-ft -n semseg-sonata-v1m1-0-base-0c-scannet-ft -w exp/sonata/pretrain-sonata-v1m1-0-base/model/model_last.pth
```

I use batch_size=64 and learning_rate=0.001(learning rate can be changed [here](https://github.com/Pointcept/Pointcept/blob/main/configs/sonata/semseg-sonata-v1m1-0c-scannet-ft.py#L55)):
```
optimizer = dict(type="AdamW", lr=0.001, weight_decay=0.02)
scheduler = dict(
    type="OneCycleLR",
    max_lr=[0.001, 0.0001],
    pct_start=0.05,
    anneal_strategy="cos",
    div_factor=10.0,
    final_div_factor=1000.0,
)
param_dicts = [dict(keyword="block", lr=0.0001)]
```

**Comment by bkxmw** [comment]
@Yujia-Zhang0913 Thank you for the detailed explanation! I have successfully reproduced the results according to the configuration instructions you provided. Excellent work!

**Comment by haomengz** [comment]
Hi @bkxmw, I am also trying to reproduce the Concerto fine-tuning results on ScanNet. May I know whether you achieve the same results as reported in the paper? What is your final re-produced results? My reproduced results are in [#548](https://github.com/Pointcept/Pointcept/issues/548#issuecomment-3624027985).
